cdap-ui/app/features/workflows/controllers/tabs/schedule-ctrl.js

```diff
@@ -0,0 +1,56 @@
+angular.module(PKG.name + '.feature.workflows')
+  .controller('WorkflowsSchedulesController', function($scope, MyDataSource, $state) {
+    var dataSrc = new MyDataSource($scope);
+
+
+    dataSrc.request({
+      _cdapNsPath: '/apps/' + $state.params.appId +
+                    '/workflows/' + $state.params.programId + '/schedules'
+    })
+    .then(function (res) {
+      $scope.schedules = res;
+
+      angular.forEach($scope.schedules, function(v) {
+        if (v.scheduleType === 'TIME') {
+          console.log('test');
+          var parse = v.schedule.cronExpression.split(' ');
+          v.time = {};
+          v.time.min = parse[0];
+          v.time.hour = parse[1];
+          v.time.day = parse[2];
+          v.time.month = parse[3];
+          v.time.week = parse[4];
```

Comment:
Are we sure that we get all 5 parse cron expressions all the time? Should we check before accessing them?